pkg/runner/local_infra.go

```diff
@@ -0,0 +1,130 @@
+package runner
+
+import (
+	"context"
+	"github.com/docker/docker/client"
+	"github.com/ipfs/testground/pkg/rpc"
```

Comment:
Import order, in groups:

- stdlib
- testground
- third parties

https://github.com/golang/go/wiki/CodeReviewComments#imports